### ISSUE TITLE
Tune debug accessibility

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#3009]
- [x] After pressing F5 to launch the app under the debugger the debugger should announce "Debugger launched" before the app starts up and the screen reader starts announcing from the app.
- [x] When a breakpoint is hit, we should announce "breakpoint hit filename linenumber". If it's a conditional breakpoint, announce "conditional breakpoint hit filename linenumber". We discussed whether or not to announce the conditional expression but thought that it might be too verbose to do so.
- [ ] Each time execution moves from one breakpoint to another, or each time a breakpoint is hit, there is a lot of text announced. The main window title is announced, followed by the file name. Then "Text file editor, press Alt+F1 for more info". We don't need to announce this each time and should focus on announcing that a breakpoint was hit.
- [x] Inside the debug viewlet, when browsing the list of variables we should reduce the verbosity as much as possible. For example, consider just announcing the name of the variable or field rather than prefixing it with "variable"

**Comment by isidorn** [comment]
Point 3 is extracted in a seperate issue #3262

**Comment by isidorn** [comment]
Giving to you Joao since you had it on the test plan

**Comment by joaomoreno** [comment]
@isidorn there is a leftover checkbox
